CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changes by Version
 ==================
 
+1.38.0 (2022-09-19)
+-------------------
+* Set alias enable variable for spark cronjob ([#2061](https://github.com/jaegertracing/jaeger-operator/pull/2061), [@miyunari](https://github.com/miyunari))
```

Comment:
note i added https://github.com/jaegertracing/jaeger-operator/pull/2066 this morning.